src/Storefront/Resources/views/storefront/layout/breadcrumb.html.twig

```diff
@@ -48,13 +48,16 @@
                             </li>
                         {% endblock %}
 
-                        {% block layout_breadcrumb_placeholder %}
-                            {% if key != breadcrumbKeys|last %}
-                                <div class="breadcrumb-placeholder">
-                                    {% sw_icon 'arrow-medium-right' style { size: 'fluid', pack: 'solid' } %}
-                                </div>
-                            {% endif %}
-                        {% endblock %}
+                        {% if not feature('v6.8.0.0') %}
```

Comment:
Hey @aragon999 👋 

thank you for your contribution. 👍 

I think this is a totally valid change and the `<div>` shouldn't be part of the list.
Overall, I would tend to only use the Bootstrap structure and use `--bs-breadcrumb-divider` with plain `>` or a base64 svg as the divider. 

What do you think about directly removing the contents of `layout_breadcrumb_placeholder ` but keep the block as deprecated? Could be a behaviour break because we have the Bootstrap divider, and if someone extends the divider, we have eventually 2 dividers at the same time. 🤔

Reply:
We could even keep the block inside a variable and pass this to `--bs-breadcrumb-divider`. But this wouldn't work because you cannot pass a complete html/svg. So probably a new block with a proper deprecation notice how to use it would be appropriate. 

```twig
{% set divider %}
    {% block layout_breadcrumb_placeholder %}
        url(&#34;data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='8' height='8'%3E%3Cpath d='M2.5 0L1 1.5 3.5 4 1 6.5 2.5 8l4-4-4-4z' fill='%236c757d'/%3E%3C/svg%3E&#34;); 
    {% endblock %}
{% endset %}

<nav style="--bs-breadcrumb-divider: {{ divider }}" >

```

Reply:
Yes this was what I was also mentioning in my comment that there could be two dividers then. I think we can keep the feature toggle and for the current version just change the `<div>` to `<li>`